packages/smoldot-test-utils/src/index.ts

```diff
@@ -88,8 +90,9 @@ export const mockSmoldot = (responder: RpcResponder, healthResponder = healthyRe
       return Promise.resolve({
         terminate: () => {},
         // fake the async reply by using the reponder to format
-        // a reply via options.json_rpc_callback
-        send_json_rpc: fakeRpcSend(options, responder, healthResponder)
+        // a reply via options.jsonRpcCallback
+        sendJsonRpc: fakeRpcSend(options, responder, healthResponder),
```

Comment:
Misses a parameter?

Reply:
These are mocks - not the actual ones

Reply:
The parameter is actually missing on line 67. `fakeRpcSend` return a function - that returned function needs the new parameter